src/compile/mark/rule.ts

```diff
@@ -27,8 +27,8 @@ export const rule: MarkCompiler = {
         range: orient !== 'vertical' // include x2 for horizontal or line segment rule
       }),
       ...encode.pointOrRangePosition('y', model, {
-        defaultPos: orient === 'vertical' ? 'zeroOrMin' : 'mid',
-        defaultPos2: 'zeroOrMax',
+        defaultPos: orient === 'vertical' ? 'zeroOrMax' : 'mid',
```

Comment:
Why do we need to swap this? and only for Y, not X?

Reply:
We should swap those as well

Reply:
Ok, but that still doesn't answer why we need to swap.  I'm sure discover an example where it matters?